uniffi_bindgen/src/bindings/python/templates/ErrorTemplate.py

```diff
@@ -0,0 +1,51 @@
+class RustError(ctypes.Structure):
+    _fields_ = [
+        ("code", ctypes.c_int32),
+        ("message", ctypes.c_char_p),
+    ]
+
+    def __str__(self):
+        return "RustError(code={}, message={})".format(self.code, str(self.message, "utf-8")) 
+
+{% for e in ci.iter_error_definitions() %}
+class {{e.name()}}:
```

Comment:
minor point, but: please use `{{ e.name()|class_name_py }}` and similar for `e.value`, to make sure these have the right case convention for python names. (Some of the other python classes don't do this either, but they should; I've tried to fix some of them over in #214

Reply:
@rfk What `|class_name_py` does?

Reply:
It basically just makes sure the names are `CamelCase`, which is the preferred naming scheme for classes in python.